app/src/main/java/com/github/se/polyfit/model/post/Post.kt

```diff
@@ -75,9 +79,17 @@ data class Post(
         val description = data["description"] as? String ?: ""
         val location = Location.deserialize(data["location"] as Map<String, Any>)
         val meal = Meal.deserialize(data["meal"] as Map<String, Any>)
-        val createdAt = deserializeLocalDate(data, "createdAt") ?: LocalDate.now()
+        val createdAt = deserializeLocalDate(data, "createdAt")
+        var imageDownloadURL: Uri? = null
 
-        val newPost = Post(userId, description, location, meal, createdAt)
+        if (data["imageDownloadURL"] is String &&
```

Comment:
If we are storing it as either `""` or the string, wouldn't this always be true, since in both case it is a string

Reply:
Good catch !